changelog/README.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 6th September 2024
+* **Breaking:** `ReleasedUtc` is no longer required in the response (change introduced on 8th January 2024 with [rolling release offsets](https://releases.mews.com/en/introducing-rolling-release-dates-to-unlock-allotments-for-availability-blocks)).

```

Comment:
As this is the first line in the Changelog entry, it doesn't make sense without specifying which operations it is referring to when it says "...is no longer required in the response".

Reply:
Changed to:

```
* **Breaking:** In [Availability block response object](../operations/availabilityblocks.md#availability-block) `ReleasedUtc` property is no longer required. The breaking change was introduced on 8th of January 2024 with [rolling release offsets](https://releases.mews.com/en/introducing-rolling-release-dates-to-unlock-allotments-for-availability-blocks). The following operations are affected:
  * [Get all availability blocks](../operations/availabilityblocks.md#get-all-availability-blocks)
  * [Update availability blocks](../operations/availabilityblocks.md#update-availability-blocks)
```

I omitted add operation since you need to explicitly opt-in to get null on that property from there.